src/senaite/core/content/senaitesetup.py

```diff
@@ -171,4 +171,5 @@ def setSiteLogoCSS(self, value):
         """Set the site logo
         """
         mutator = self.mutator("site_logo_css")
-        return mutator(self, value)
```

Comment:
This line break got lost in https://github.com/senaite/senaite.core/pull/2075/files#diff-8cb942695b5f35c916b14bb3127f39efba58b20b8d31598e92cb62c8f0a15682 and resulted in flake8 failure: https://app.travis-ci.com/github/senaite/senaite.core/builds/253849297